sd-miner-v0.0.3.py

```diff
@@ -319,8 +319,8 @@ def execute_model(config, model_id, prompt, neg_prompt, height, width, num_itera
         config.loaded_models[model_id] = current_model
 
     kwargs = {
-        'height': min(height, config.config['general']['max_height']),
-        'width': min(width, config.config['general']['max_width']),
+        'height': min(height, config.config['general']['max_height'] - config.config['general']['max_height'] % 8),
```

Comment:
if `height` is not divisible by 8, this change does not take effect.

I think we need to use `height - height % 8` not `max_height`

Reply:
width/height = 0 can be ignored. these are filtered out on the server side

Reply:
Good catch, fixed that